src/main/java/org/numenta/nupic/SDR.java

```diff
@@ -85,9 +85,7 @@
      * @return  the column indexes of the specified cells.
      */
     public static int[] cellsToColumns(List<Cell> cells, int cellsPerColumn) {
-        IntStream op = cells.stream().mapToInt(c -> c.getIndex());
-            
-        return op.map(cellIdx -> cellIdx / cellsPerColumn).distinct().toArray();
+        return cells.stream().mapToInt(c -> c.getColumn().getIndex()).distinct().toArray();
     }
```

Comment:
This looks good. Removing a temporary variable on the heap is a good thing and Cells have a reference to their Columns so the simplification is great. I would remove the ```cellsPerColumn``` parameter passed in since it isn't being used.